proposed/kubeflow/v0.6.2/questions.yml

```diff
@@ -0,0 +1,158 @@
+categories:
+- machine learning
+rancher_max_version: 2.3.3
+labels:
+  io.rancher.certified: experimental
+namespace: kubeflow
+questions:
+- variable: profiles.admin
+  default: johnDoe@acme.com
+  description: "Kubeflow admin"
+  type: string
+  label: Admin
+- variable: applicationController.project
+  default: "test"
+  description: "Application controller project"
+  type: string
+  label: Application controller project
+- variable: argo.executorImage
+  default: "argoproj/argoexec:v2.3.0"
+  description: "Argo executor image"
+  type: string
+  label: Argo executor image
+- variable: argo.artifactRepositoryBucket
+  default: "mlpipeline"
+  description: "Argo artifact repository bucket"
+  type: string
+  label: Argo artifact repository bucket
+- variable: argo.artifactRepositoryKeyPrefix
+  default: "artifacts"
+  description: "Argo artifact repository key prefix"
+  type: string
+  label: Argo artifact repository key prefix
+- variable: argo.artifactRepositoryAccessKeySecretName
+  default: "mlpipeline-minio-artifact"
+  description: "Argo artifact repository access key name"
+  type: string
+  label: Argo artifact repository access key name
+- variable: argo.artifactRepositoryAccessKeySecretKey
+  default: "accesskey"
+  description: "Argo artifact repository access key secret"
+  type: string
+  label: Argo artifact repository access key secret
+- variable: argo.artifactRepositorySecretKeySecretName
+  default: "mlpipeline-minio-artifact"
+  description: "Argo artifact repository secret key name"
+  type: string
+  label: Argo artifact repository secret key name
+- variable: argo.artifactRepositorySecretKeySecretKey
+  default: "secretkey"
```

Comment:
are those `secretkey` and `acceskey` exist? if not and when those variables are not required would prefer to leave it as an empty string, also the `questions.yml` support variable type `secret`.

Reply:
Ok, updated to type `password`